var/www/html/tools/MemcachedAdmin/Library/Analysis.php

```diff
@@ -0,0 +1,332 @@
+<?php
+/**
+ * Copyright 2010 Cyrille Mahieux
```

Comment:
...actually, is it best to put this in the repo (essentially forking it), or should it be installed via the Dockerfile?